components/section/Project.jsx

```diff
@@ -0,0 +1,96 @@
+import PageLink from "../ui/Link";
+import Svg from "../ui/Svg";
+import React, { useState } from "react";
+import ProjectCard from "../cards/ProjectCard";
+import Wrapper from "../wrappers/Wrapper";
+export default function Project() {
+  const [show, setShow] = useState(null);
+  const projects = [
+    "Dacade",
+    "Bitlipa",
+    "Symplifi",
+    "UTU.IO",
+    "NeueUX",
+    "Lab3",
+    "Even"
+  ];
+
+  const images = [
+    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/dacade.webp", text: "Peer-to-Peer learning platform" },
+    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/bitlipa.webp", text: "Peer-to-Peer learning platform" },
+    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/symplifi.webp", text: "Peer-to-Peer learning platform" },
+    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/utu.webp", text: "Peer-to-Peer learning platform" },
+    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/neueux.webp", text: "Peer-to-Peer learning platform" },
+    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/lab3.webp", text: "Peer-to-Peer learning platform" },
+    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/even.webp", text: "Peer-to-Peer learning platform" },
+  ];
+
+  return (
+   <Wrapper>
+     <section className="flex flex-col xl:flex xl:flex-row justify-center sm:mt-auto mt-32">
+      <div className="flex w-full justify-between">
+        <ul className="font-Neufile xl:w-59.25">
+          <h3 className="mb-2.75 text-dark-grey md:text-2xl text-xl">Project</h3>
+          {projects.map((item, index) => (
+            <li
+              className="md:h-18  group mb-0.5  -ml-2 flex flex-col justify-center leading-18 tracking-spacing-xl"
+              key={index}
+            >
+              <div className="group w-fit mb-7 md:mb-1">
+                <PageLink
+                  onClick={() => SetMobileImage("flex")}
+                  style={
+                    "md:gap-4 gap-0.5 md:pt-auto pt-5 border-b-2 border-white hover:border-b-2 hover:border-transparent -pl-2 ml-2 pb-3"
+                  }
```

Comment:
This function doesn't seem to be imported or defined anywhere. does it work? if it does you should rename it to something descriptive if it doesn't work then you should remove it